spec/compiler/parser/parser_spec.cr

```diff
@@ -1650,7 +1660,7 @@ module Crystal
 
     assert_syntax_error "def foo(var : Foo+); end"
 
-    %w(&& || !).each do |name|
```

Comment:
This change should only remove `!`, not `||`. This is a string array literal, not a logic expression ;)

We could however consider allowing regular call syntax for `&&` and `||` as well, and remove this section entirely.

Reply:
Oh, thank you, good eyes! You have a good point.